src/org/omegat/gui/preferences/view/GeneralOptionsController.java

```diff
@@ -84,13 +84,13 @@ private void openFile(File path) {
     @Override
     protected void initFromPrefs() {
         panel.tabAdvanceCheckBox.setSelected(Core.getEditor().getSettings().isUseTabForAdvance());
-        panel.confirmQuitCheckBox.setSelected(Preferences.isPreference(Preferences.ALWAYS_CONFIRM_QUIT));
+        panel.confirmQuitCheckBox.setSelected(!Preferences.isPreference(Preferences.ALWAYS_CONFIRM_QUIT));
```

Comment:
It should not be reverted, that become a logic bug. 

```suggestion
        panel.confirmQuitCheckBox.setSelected(Preferences.isPreferenceDefault(Preferences.ALWAYS_CONFIRM_QUIT, true));
```